### ISSUE TITLE
h package

### ISSUE THREAD

**Issue by shogg** [maddalax/htmgo#10]
In the examples I saw you use the h-package. What is this naming? Don't burn commonly used symbols. Call it htmgo or something like that.

**Comment by maddalax** [comment]
What are you concerned about specifically? If you need to use 'h', you could alias it 

aliasing it 
```
htmgo "github.com/maddalax/htmgo/framework/h"
```

```
. "github.com/maddalax/htmgo/framework/h"
```

**Comment by shogg** [comment]
I didn't know the h-package is for creating markup. Is it part of a markup language?
"Too verbose package name forces everyone to alias it by default." It forces you or it gives you the choice to leave the long name. With a one letter package name you're more often forced to alias it.
This was my last comment. You can close the issue as you like.

**Comment by maddalax** [comment]
> I didn't know the h-package is for creating markup. Is it part of a markup language? 

Yep, the h package in htmgo includes all the tags and attributes
```
h.Div
h.Button
h.Id
h.Class
etc...
```

> With a one letter package name you're more often forced to alias it.

Appreciate the feedback but I'm not totally convinced its common enough to have one letter variable names such as 'h' to facilitate needing to alias very often.

Even if I was writing a triple for loop using single letter variable names, I would use the standard `ijk`

I'll keep this issue open just in case there are more arguments in favor of renaming it, but this is where I stand on it at the moment.
